openedx/core/lib/session_serializers.py

```diff
@@ -0,0 +1,12 @@
+from django.contrib.sessions.serializers import PickleSerializer
+
+
+class PickleV2Serializer(PickleSerializer):
```

Comment:
[Source for the PickleSerializer](https://github.com/django/django/blob/37835883ad55b3c4c3340eb8721b41fffe3ee0ef/django/contrib/sessions/serializers.py)